framework/src/runtime.rs

```diff
@@ -3,18 +3,23 @@ use common::Result;
 use config::{NetBricksConfiguration, CLI_ARGS};
```

Comment:
I know we test the runtime out via examples, but maybe we want some simple unit tests around it, including sending signals, using a timer, etc...? Not a biggie, but thinking aloud on it.

Reply:
I'm not sure these things (unix signals, timer behavior, nb context, process exit code) are easily unit testable.

Reply:
Figured